bin/add-mod-path.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+def error(*args):
+    print(*args)
+    sys.exit(1)
+
+def info():
+    print(f"""This program, {sys.argv[0]} provides
+a convenient way to create modules for the environment module for
+BOUT++ that sets PYTHONPATH, PATH and BOUT_TOP, so that switching
+between different BOUT++ installations is easy.
+{sys.argv[0]} stores the loaded modules as prerequesit, so that you
```

Comment:
`prerequesit` -> `prerequisites`

(It literally always takes me two attempts to type this word!)